### ISSUE TITLE
Add checks if terminal text colors stay readable on the background.

### ISSUE THREAD

**Issue by ciembor** [ciembor/4bit#55]
Use WCAG 2.x contrast: PASS >= 4.5:1, WARN >= 3.0:1, FAIL < 3.0:1.

**Comment by ciembor** [comment]
Done in https://github.com/ciembor/4bit/commit/0e213bc797b003da1d40190327dad9dfa18b6679
